src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -501,6 +511,14 @@ public interface IStartConfiguration
             /// No update checks or analytics collection should be done.
             /// </summary>
             bool IsHeadless { get; set; }
+            /// <summary>
+            /// Default Python script engine
+            /// </summary>
+            string DefaultPythonEngine { get; set; }
+            /// <summary>
+            /// Provide values for startup options
+            /// </summary>
+            //IDictionary<string, string> StartupOptions { get; set; }
```

Comment:
This is like an alternative API I had in mind at first. It would allow to provide command-line options in a generic way. The only benefit I can think about it is that we don't add an explicit Python property to the config. On the other hand, it is messier to add entries to a dictionary than assigning properties.

Reply:
I've always thought it would be nice to be able to override the entire config with an xml or json file - but thats a discussion for another time I think.

Reply:
We already have multiple, partially overlapping systems to manage config, with StartConfiguration, IPreferences, StartupUtils.CommandLineArguments, properties copied into models, etc. Rationalizing all of this should be a priority for the next API-breaking release, would love someone to research and draft a proposal.